features/support/webmock.rb

```diff
@@ -8,7 +8,7 @@
 solr_host = ENV['TEST_SOLR_HOST'] || 'localhost'
 solr_port = ENV['TEST_SOLR_PORT'] || 8981
 
-WebMock.disable_net_connect!(   :allow_localhost => true, 
-                                :allow => [ "#{solr_host}:#{solr_port}",
-                                            "codeclimate.com" ] 
-                            )
+WebMock.disable_net_connect!(allow_localhost: true,
+                             allow: ["#{solr_host}:#{solr_port}",
+                                     'codeclimate.com',
+                                     'host.docker.internal:9515'])
```

Comment:
Can you add a comment about what is this port used for? 
Is that a way to inspect info about the host the container is running on?

Reply:
When running features in non-headless mode, `chromedriver` must be running on the host machine at port `9515` (the default port). Webmock will block this connection by default, so we must explicitly allow it.